js/atoms/src/Empty.svelte

```diff
@@ -22,6 +28,14 @@
 		color: var(--body-text-color);
 	}
 
+	.small {
+		min-height: calc(var(--size-32) - 20px);
+	}
+
+	.large {
+		min-height: calc(var(--size-64) - 20px);
+	}
+
```

Comment:
@aliabid94 This is the fix for the icon alignment. These were `height` previously, so we were getting both a `min-height` _and_ a `height` which was the wrong behaviour. Because this was a height it was overwriting the 100% height (on line 16) which makes the icon alignment work with whatever height the parent is set to (custom or otherwise).